R/demo-education.R

```diff
@@ -1,20 +1,32 @@
 #' Access to Education Data
 #'
-#' This function returns a data frame from Education in State Cancer Profiles
+#' This function returns a data frame about education demographics from State Cancer Profiles.
 #'
 #' @param area A state/territory abbreviation or USA.
-#' @param areatype Either "county", "hsa" (Health service area), or "state"
-#' @param education Either "less than 9th grade", "at least high school", "at least bachelors degree"
-#' @param sex Either "both sexes", "male", "female"
-#' @param race One of the following values: "All Races (includes Hispanic)", "white (includes hispanic)",
-#'              "white non-hispanic","black","amer. indian/alaskan native (includes hispanic)",
-#'              "asian or pacific islander (includes hispanic)","hispanic (any race)
+#' @param areatype Either `"county"`, `"hsa"` (Health Service Area), or
```

Comment:
I think the important thing is to pick a formatting choice and then to be consistent throughout the documentation. Something I like about using backticks is that is makes the text on the page look like code, which I feel like is a visual signal that the user can copy and paste it into their code. None of our potential users want to type out `"Amer. Indian/Alaskan Native (includes Hispanic)"`. I think my vote is that anything that plausibly could be copied out of the docs and into the code should be surrounded by backticks.

Reply:
great, I completely agree, I will make sure that all of the params have backticks